packages/webpack-plugin/src/webpack-config-stylable-excludes.ts

```diff
@@ -54,3 +54,31 @@ function safelyWalkJSON(
         }
     }
 }
+
+export function bundleLibs(config: Configuration, packages: string[], isServer: boolean) {
+    if (isServer) {
```

Comment:
It's weird to call `bundleLibs` and for some reason it will only work on the server. 
So maybe change the name to `bundleLibsOnServerToPreventNodeImports`?

Reply:
Changed to `bundleServerLibs`